modules/apps/adaptive-media/adaptive-media-image-item-selector-impl/src/main/java/com/liferay/adaptive/media/image/item/selector/internal/FileEntryAMImageFileEntryItemSelectorReturnTypeResolver.java

```diff
@@ -63,9 +63,8 @@ public String getValue(FileEntry fileEntry, ThemeDisplay themeDisplay)
 		String previewURL = null;
 
 		if (fileEntry.getGroupId() == fileEntry.getRepositoryId()) {
-			previewURL = DLUtil.getPreviewURL(
-				fileEntry, fileEntry.getFileVersion(), themeDisplay,
-				StringPool.BLANK, false, false);
+			previewURL = DLUtil.getImagePreviewURL(
```

Comment:
This change seems to be incorrect, since in the previous method call (getPreviewURL) we are not appending the version and the url is not absolute.

I think that you should use the same method as we did with "&imagePreview=1" as previewQueryString or create a new method "getImagePreviewURL" that receives the appendVersion and absoluteURL parameters to false.

Please resend with this changes.

Reply:
Thanks for pointing that out. I'll resend the pr with the changes.